docs/Developers/Newcomers_Guide.rst

```diff
@@ -363,3 +363,51 @@ go for `a low issue <https://coala.io/low>`__ - they are really rewarding!
     the solution in our :doc:`useful links section <Useful_Links>`.
 
 .. _this page: https://docs.coala.io/en/latest/Help/FAQ.html#what-are-those-things-failing-passing-on-my-pull-request
+
+Step 9. Finishing the newcomer process
+--------------------------------------
+
+So far you should have covered three of the five steps in the newcomer process:
+
+  - run coala on a project of yours and fill our
+    `usability survey <http://coala.io/usability>`_.
+  - merge a ``difficulty/newcomer`` Pull Request and fill in our
+    `survey form <http://coala.io/newform>`_.
+  - review at least a ``difficulty/newcomer`` Pull Request.
+
+This means your are only missing the last two steps:
+
+  - merge a ``difficulty/low`` Pull Request.
+  - review at least a ``difficulty/low`` or higher Pull Request.
+
+The process is the same as for the `newcomer` issues and pull requests.
+
+  - Find a `difficulty/low` issue at http://coala.io/low
```

Comment:
I'm not even sure if rst automatically links them, didn't see anything in the docs, can we have proper links? Also not showing the URL directly to the user might be nice